Library/Homebrew/cmd/list.rb

```diff
@@ -20,13 +20,23 @@
 
 require "metafiles"
 require "formula"
+require "cli_parser"
 
 module Homebrew
   module_function
 
   def list
+    @args = CLI::Parser.parse do
+      switch "--unbrewed"
+      switch "--pinned"
+      switch "--versions"
+      switch "--full-name"
+      switch "--multiple", required_for: "--versions"
```

Comment:
Should be `depends_on`.

Reply:
So, what does the phrase
```
`--multiple` requires `--versions`
```
in plain English corresponds to in this new syntax?
```
switch "--multiple", depends_on: "--versions"
```
?

Reply:
```ruby
switch "--multiple", depends_on: "--versions"
```

`--multiple` is optional and can only be given as a “sub-option” for `--versions`.


```ruby
switch "--multiple", required_for: "--versions"
```

When `--versions` is given, `--multiple` also has to be specified.